akka-docs/src/main/paradox/cluster-routing.md

```diff
@@ -148,7 +148,7 @@ The easiest way to run **Router Example with Group of Routees** example yourself
 @java[@extref[Akka Cluster Sample with Java](ecs:akka-samples-cluster-java)]
 together with the tutorial. It contains instructions on how to run the **Router Example with Group of Routees** sample.
 The source code of this sample can be found in the
-@scala[@extref[Akka Samples Repository](samples:akka-sample-cluster-scala)]@java[@extref[Akka Samples Repository](samples:akka-sample-cluster-java)].
+@scala[@extref[Akka Samples Repository](samples:akka-samples-cluster-scala)]@java[@extref[Akka Samples Repository](samples:akka-samples-cluster-java)].
```

Comment:
I think the originally intended link here was to https://github.com/akka/akka-samples/tree/2.5/akka-sample-cluster-scala

It was changed in https://github.com/akka/akka/commit/91cbd6ed99c30b5179f24519de50009c863d5063, but without changing all usages.

Here we link to both a download with `extref ecs` and to the source. I suggest that we change this to only link to `extref samples`, something like this

```
The easiest way to run **Router Example with Group of Routees** example yourself is to try the
@scala[@extref[Akka Cluster Sample with Scala](samples:akka-samples-cluster-scala)]@java[@extref[Akka Cluster Sample with Java](samples:akka-samples-cluster-java)].
It contains instructions on how to run the **Router Example with Group of Routees** sample.
```

Same kind of change at other places.

Reply:
@patriknw you want to remove the repo links/text?